scipy/optimize/__init__.py

```diff
@@ -97,7 +97,8 @@
 
    basinhopping - Basinhopping stochastic optimizer
    brute - Brute force searching optimizer
-   differential_evolution - stochastic minimization using differential evolution
+   DifferentialEvolution - stochastic minimization using differential evolution
+   differential_evolution - convenience function for differential evolution
```

Comment:
These names/descriptions may be a bit confusing. I don't have a concrete suggestion, but it would be good to indicate to the reader that they should use ``DifferentialEvolution`` only if they want some kind of customization of ``differential_evolution`` behavior beyond what the keywords of that function provide.